dev/ProgressBar/ProgressBar.cpp

```diff
@@ -0,0 +1,171 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "common.h"
+#include "ProgressBar.h"
+#include "RuntimeProfiler.h"
+#include "ResourceAccessor.h"
+
+ProgressBar::ProgressBar()
+{
+    __RP_Marker_ClassById(RuntimeProfiler::ProfId_ProgressBar);
+
+    SetDefaultStyleKey(this);
+
+    m_sizeChangedRevoker = SizeChanged(winrt::auto_revoke, { this, &ProgressBar::OnSizeChanged });
```

Comment:
As @jevansa@microsoft.com mentioned in Tammy's PR, you don't need revokers when you are registering to your own events so this one could be removed